Extension/lib/content-script/preload.js

```diff
@@ -89,7 +89,7 @@ var PreloadHelper = {
         } else {
             this._applySelectors(response.selectors);
             this._applyScripts(response.scripts);
-            if (this.collapseAllElements) {
+            if (this.collapseAllElements || response.allSelectorsLoaded === true) {

```

Comment:
At least let's rename it to show that the only place where we need it is Safari 9+